### DISCUSSION TITLE
Format vs informal style in translations

### DISCUSSION THREAD

**Discussion by mgeisler** [google/comprehensive-rust#818] [General]
Hi all,

A [question](https://github.com/google/comprehensive-rust/pull/805#discussion_r1228899020) was brought up by @markx for the Simplified Chinese translation: should we use a formal or orthodox style in the translations or instead use a more informat or laid-back style?

My personal style would be an informal style. As an example, the German translation #285 agreed to use the informal _du_ instead of the format _Sie_ when translating the English 1st person _you_ (@Throvn and @ronaldfw, please correct me if I'm wrong). Similarly, I tried to use a friendly and informal style when writing the English text.

**Comment by henrif75** [comment]
A little late here, but I agree with @mgeisler. Using an informal style makes the course more "approachable".

**Reply by mgeisler** [reply]
Good point, I hadn't thought of it from that angle :slightly_smiling_face: Can I get you to add some language about this to the [style guide](https://github.com/google/comprehensive-rust/blob/main/STYLE.md)?

**Reply by henrif75** [reply]
I've created #1185 to track this subject.

**Comment by un-variable** [comment]
Hey! 
It's important to consider the appropriate style for translations, as it can greatly impact how a message is perceived. Using a formal or informal style depends on the context and the tone you want to convey.
 In your example, opting for an informal style can make the communication feel more friendly and approachable. However, it's essential to maintain consistency within the translation project and ensure it aligns with the project's goals and audience.

**Comment by detro** [comment]
Hi!
In regards to English, an "informal+technical" style works best for things like "approachability", "tone" and "correctness". I feel that the industry, being heavily weighted towards using English in most contexts, has organically developed a style where a little humour, and not taking itself too seriously works best.

In regards to Italian, keeping it informal is important but a bit harder. Italian can lead to long form sentences, and excessive use of pronouns, that can lead almost to "legalese". So, a conscious effort is required to "keep things friendly".

**Reply by un-variable** [reply]
Hi  @detro ;
Your insights on the "informal+technical" style in English are spot on. It strikes a balance between approachability, tone, and correctness, aligning with the industry's tendency to use English in various contexts.
 A touch of humor and not taking things too seriously can indeed work wonders.

For Italian, I understand the challenge of avoiding excessive formality and lengthy sentences. Keeping the language friendly and approachable while being mindful of potential legalese tendencies is a great approach. 
It's all about making the content accessible and engaging for the audience.

**Comment by hueich** [comment]
Late to the party, but here are my thoughts.  For the Traditional Chinese version, using the common informal style (你 vs 您) and keeping sentences technical and concise and not too colloquial or wordy would be a good approach.  I think keeping the little humor and lightheartedness from the English is great, but not critical if it doesn't translate well.  From my understanding, if this repo is meant to be used for teaching, then the teacher can add their own personality and style to the material that's appropriate to the audience, so the material itself doesn't need to try too hard to be funny or lighthearted.

**Reply by un-variable** [reply]
Thank you for sharing your thoughts on the Traditional Chinese version. 
Your suggestions make sense - using common informal style while keeping the content technical and concise is a good approach. Maintaining some humor and lightheartedness is a bonus but not essential. It's indeed a good point that teachers can add their own personality to the material for their specific audience.
